brave/ledger-publisher
# Segment top100 publishers into categories

- Move publishers from top100 list into separate categories
- Remove top100 category

Auditor: @mrose17

Test Plan:
- Run `npm run build-rules`
- Ensure rules from existing and new categories are built correctly

Fixes: #14
